src/components/feed-sources/feed-source-type-templates/EventDatabaseApiFeedType.template.jsx

```diff
@@ -0,0 +1,37 @@
+import { React } from "react";
+import PropTypes from "prop-types";
+import FormInput from "../../util/forms/form-input";
+
+const EventDatabaseApiTemplate = ({
+  handleInput,
+  formStateObject,
+  t,
+  mode,
+}) => {
+  return (
+    <>
+      <FormInput
+        name="host"
+        type="text"
+        label={t("dynamic-fields.EventDatabaseApiFeedType.host")}
```

Comment:
see comment above, this will allow you to do:

label={t("host")}

instead of all the dots

Reply:
Uh, nice!